.github/workflows/test.yml

```diff
@@ -22,7 +27,10 @@ jobs:
       - name: Install Node dependencies
         run: yarn --frozen-lockfile
 
-      - run: yarn lint
+      - name: lint (only on ubuntu)
+        if: ${{ matrix.os !== 'windows-latest' }}
```

Comment:
Why is linting disabled on windows?

Reply:
I couldn't figure how to resolve the CRLF issues. Since it runs on ubuntu, errors won't creep into the code base.

Reply:
I fixed it correctly in https://github.com/vega/ts-json-schema-generator/pull/663

Reply:
thank you.

Reply:
Thanks for sending the pull request.